satellite/vouchers/service.go

```diff
@@ -18,13 +18,13 @@ import (
 	"storj.io/storj/pkg/pb"
 )
 
```

Comment:
there's not really a good place to add this comment, but should this file be named endpoint.go rather than service.go now?

Reply:
oh thanks for catching that!